http/src/main/resources/reference.conf

```diff
@@ -596,6 +596,45 @@ vpc {
         }
       ]
     }
+    # Allows SSH access from google IAP proxy
+    # IP list obtained from https://cloud.google.com/iap/docs/using-tcp-forwarding#create-firewall-rule
```

Comment:
Do we need all of these? ALA are we testing in/from a single environment?

Reply:
Oh, good point. I should be able to just use the one region in the automation tests here. Let me test this out...

Reply:
Huh... ok I'm pretty confused. But we do apparently need all these (source) regions. 
```
terra-quality-eb725221/automation-test-ashspqrlz errored due to List(RuntimeError(ErrorReport(leonardo,Fail to create firewall due to unsupported Region europe-west1,Some(500 Internal Server Error),List(),List(),Some(class org.broadinstitute.dsde.workbench.leonardo.model.LeoException),Some(TraceId(3dfe4ca55a8a4e00864d380e2c15d72f/3939911508519804487))),None,2024-01-05T00:14:07Z,Some(TraceId(3dfe4ca55a8a4e00864d380e2c15d72f/3939911508519804487))))
```